### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by rek990**
## Summary
This PR includes the initial build of QSO Corner. Specifically, this includes the main page, which displays a comprehensive table of all QSOs of a given operator, and a page where an operator can enter a new QSO and see more information about said contact, such as name, address, license information, and any previous QSOs.

## Test(s)
At the opening of this PR, the app was rendering correctly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,73 @@ git clone git@github.com:rek990/qso-corner.git
 cd qso-corner
 npm start
 ```
+# Getting Started with Create React App
```
Comment (bkgreve) [inline README.md:10]: In the future, update the README to make it specific to your project

File: src/components/alerts/ErrorAlertQSO.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  const [show, setShow] = useState(true);
+
+  return (
+    <>
+      {show ? (
+        <Alert variant="danger" onClose={() => setShow(false)} dismissible>
+          <AlertTitle>You've encountered an error.</AlertTitle>
+          <AlertDescription>
+            An error occurred when loading this page. Please try refreshing the page, and if you
+            receive this message again, contact QSO Corner support at for further assistance.
```
Comment (bkgreve) [inline src/components/alerts/ErrorAlertQSO.jsx:14]: `support at for` --> `support for`

File: src/components/logbook/LogbookContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      </VStack>
+    </Flex>
+  );
+};
+
+export default LogbookContainer;
+
+{
+  /*<LogbookTableAPI />*/
+}
```
Comment (bkgreve) [inline src/components/logbook/LogbookContainer.jsx:33]: Remove if not needed

File: src/components/logbook/LogbookTable.jsx [hunk 4]
```diff
@@ -0,0 +1,405 @@
...
+// when backend is set up, pass { columns, data } as props
+
+const LogbookTable = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  const data = useMemo(
+    () => [
+      {
+        qsoNumber: 1,
+        callSign: "W4GI",
```
Comment (bkgreve) [inline src/components/logbook/LogbookTable.jsx:57]: Use made up call signs

**Review by bkgreve — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rek990) [reply README.md:10]: I updated and pushed to the main branch.

File: src/components/logbook/LogbookContainer.jsx (hunk 3 above)
Comment (rek990) [reply src/components/logbook/LogbookContainer.jsx:33]: LogbookTableAPI will be needed once the hard-coded data from the tables is removed.

**Review by bkgreve — APPROVED** [review]

**Pull request merged**